src/api_server.h

```diff
@@ -77,6 +77,8 @@ class MetadataApiServer {
                                std::shared_ptr<HttpServer::connection> conn);
   void HandleHealthz(const HttpServer::request& request,
                      std::shared_ptr<HttpServer::connection> conn);
+  void HandleMetrics(const HttpServer::request& request,
```

Comment:
Should we add tests to api_server_unittest.cc for this? (Possibly in a separate PR.)

Reply:
I follow the same pattern as other handlers, and all other handler doesn't has test yet either (probably because the test involved connection), so that's why I decided to not add test here. However, I still tested the most important function for this handler: https://github.com/Stackdriver/metadata-agent/pull/210/files#diff-db7be20cd5a1f1a10cb7d38f58c0fb35R212

funny enough, I found that I should change the header from `Content-Type: application/json` to `Content-Type: text/plain; version=0.0.4` and encoding header - which is described in https://prometheus.io/docs/instrumenting/exposition_formats/#basic-info

Reply:
updated the content-type part. won't add api_server_unittest in this PR.